src/index.js

```diff
@@ -135,75 +156,84 @@ const extendCodeBlockWithSyntaxHighlighting = ( settings ) => {
 			};
 
 			const sortedLanguageNames = sortBy(
-				Object.entries( languagesNames ).map( ( [ value, label ] ) => ( { label, value } ) ),
+				Object.entries(
+					languagesNames
+				).map( ( [ value, label ] ) => ( { label, value } ) ),
 				( languageOption ) => languageOption.label.toLowerCase()
 			);
 
 			const plainTextProps = {
 				value: attributes.content || '',
-				highlightedLines: parseSelectedLines( attributes.selectedLines ),
+				highlightedLines: parseSelectedLines(
+					attributes.selectedLines
+				),
 				onChange: ( content ) => setAttributes( { content } ),
-				placeholder: __( 'Write code…', 'syntax-highlighting-code-block' ),
-				'aria-label': __( 'Code', 'syntax-highlighting-code-block' ),
+				placeholder: __( 'Write code…' ),
+				'aria-label': __( 'Code' ),
```

Comment:
I don't know the WP API very well, but are these changes intended?

Reply:
Good question. I think actually yes because then they'll re-use the core translation strings rather than having to translate new duplicate ones.